index.html

```diff
@@ -2331,7 +2331,14 @@ <h3>
 The <a>DID method</a> specification MUST specify how a client can update a
 <a>DID document</a> on the <a>DID registry</a>, including all cryptographic
 operations necessary to establish proof of control, <em>or</em> state that
-updates are not possible.
+updates are not possible. 
+        </p>
+
+        <p>
+          An Update to a DID is any change in the data used to produce a DID Document. 
```

Comment:
```suggestion
          An update to a DID is any change in the data used to produce a DID Document. 
```

No need to capitalize 'update'.